libs/agent-sm/agent-policy/build.gradle

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+apply plugin: 'opensearch.build'
+apply plugin: 'opensearch.publish'
+
+ext {
+  failOnJavadocWarning = false
+}
+
+base {
+  archivesName = 'opensearch-agent-policy'
+}
+
+disableTasks('forbiddenApisMain')
```

Comment:
We should aim to enable this check and the failOnJavadocWarning checks for this module. 

When testing locally I get errors when removing this line:

> FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':libs:agent-sm:agent-policy:forbiddenApisMain'.
> Parsing signatures failed: Class 'joptsimple.internal.Strings' not found on classpath while parsing signature: joptsimple.internal.Strings

I'll file an issue around fixing this check